internal/sudo/contract_manager.go

```diff
@@ -75,37 +74,72 @@ type OpenAckDetails struct {
 	CounterpartyVersion   string `json:"counterparty_version"`
 }
 
+type ContractMethods interface {
+	HasContractInfo(sdk.Context, sdk.AccAddress) bool
+	Sudo(sdk.Context, sdk.AccAddress, []byte) ([]byte, error)
+}
+
+var _ ContractMethods = (*ContractManager)(nil)
```

Comment:
what's the point of this check? the ContractManager is never used as ContractMethods

Reply:
Yes you are right, it was left from previous tries, I need to alter ContractMethods method and add `NewSudoHandler` to it, this helps me to use this interface in the keepers. to pass ContractManager.